src/preamble.js

```diff
@@ -512,17 +512,19 @@ function stringToAscii(str, outPtr) {
 }
 {{{ maybeExport('stringToAscii') }}}
 
-// Given a pointer 'ptr' to a null-terminated UTF8-encoded string in the given array that contains uint8 values, returns
-// a copy of that string as a Javascript String object.
+// Given a pointer 'ptr' to an UTF8-encoded string (optionally null terminated) in the given array that contains uint8 values, 
+// returns a copy of that string as a Javascript String object.
+// len parameter (optional): bytewise length in case string is not null terminated
```

Comment:
Also the comment here refers to "in case string is not null terminated", but since this is not specific to null termination, it would be more accurate to say

    // maxBytes: Limits the number of bytes to read from the array at most. String read finishes when a null terminator is met, or when this many bytes have been read, whichever comes first. Note that partial/incomplete UTF-8 bytes at the end are discarded.